frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -105,7 +98,7 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
   } else if (logs) {
     data = logs;
   } else {
-    data = 'No content';
+    data = 'No logs available';
```

Comment:
Adhering to [Case 9 of the mocks](https://www.figma.com/proto/rJjOtl3nxpW6BAcl0MZr4y/Pipeline-log-viewer?page-id=62%3A1895&type=design&node-id=426-1918&viewport=344%2C-2019%2C0.11&t=3XnKD7cO6bo81JQW-1&scaling=min-zoom&starting-point-node-id=624%3A2327&show-proto-sidebar=1)